app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def flip_coin(cases):
+    dictionary = {i: 0 for i in range(11)}
+
+    for _ in range(cases):
+        heads = 0
+        for _ in range(10):
+            if random.randint(1, 2) == 1:
+                heads += 1
+        dictionary[heads] = dictionary.get(heads, 0) + 1
```

Comment:
If you used get() method you dont need dict comprehension when declare dictionary

Reply:
So in that case you may write dictionary[heads] += 1, without get method. Generaly it's not mistake or problem, just a little advice)

Reply:
oh, i see now, thanks)